src/cli_plugin/cli.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
+import pkg from '../utils/packageJson';
+import Command from '../cli/Command';
+import listCommand from './list';
+import installCommand from './install';
+import removeCommand from './remove';
+
+let argv = process.env.kbnWorkerArgv ? JSON.parse(process.env.kbnWorkerArgv) : process.argv.slice();
+let program = new Command('bin/kibana-plugin');
```

Comment:
The focus of my changes on this file were removing existing functionality from it. I didn't rewrite this file, so I didn't focus on it as a whole. Despite this, I'll make your suggested change.